plans/logs.tf

```diff
@@ -26,6 +26,14 @@ resource "azurerm_storage_share" "logs" {
     depends_on              = ["azurerm_resource_group.logs","azurerm_storage_account.logs"]
 }
 
+resource "azurerm_storage_share" "accountapp" {
```

Comment:
Why is this in the logs plan? I think this should be in a plan all to itself for the account app in case we need to provision new resources later.